operations/companies.md

```diff
@@ -47,152 +50,231 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
-| `ElectronicInvoiceIdentifier` | string | optional | Electronic invoice identifier of the company. |
-| `InvoiceDueInterval` | string | optional | The maximum time, when the invoice has to be be paid in ISO 8601 duration format. |
-| `AccountingCode` | string | optional | Accounting code of the company. |
-| `MotherCompanyId` | guid | optional | Unique identifier of mother company. |
-| `CreatedUtc` | string | optional | Date of [Company](#company) creation date and time. |
-| `UpdatedUtc` | string | optional | Date of [Company](#company) last update date and time. |
-| `AddressId` | string | optional | Unique identifier of the company [Address](addresses.md#account-address). |
-| `BillingCode` | string | optional | Billing code of the company. |
-| `Iata` | string | optional | Iata of the company. |
-| `Telephone` | string | optional | Contact telephone number. |
-| `InvoicingEmail` | string | optional | Email for issuing invoices to the company. |
-| `ContactPerson` | string | optional | Contact person of the company. |
-| `Contact` | string | optional | Other contact details, such as telephone, email or similar. |
-| `Notes` | string | optional | Additional notes. |
-| `Options` | [Company options](#company-options) | required | Options of the company. |
-| `Department` | string | optional | The internal segmentation of a company, e.g. sales department. |
-| `DunsNumber` | string | optional | The Dun & Bradstreet unique 9-digit DUNS number. |
-| `CreditRating` | [Credit rating](#credit-rating) | optional | Credit rating to define creditworthiness of the company. |
-| `ReferenceIdentifier` | string | optional | External system identifier - custom identifier used by an external system such as an external database. |
-| `WebsiteUrl` | string | optional | The website url of the company. |
-| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of company from external system. |
+> ### Deprecated!
+> This operation is [deprecated](../deprecations/README.md). 
 
-#### Company options
+### Request
 
-| Property | Type | Contract | Description |
-| :-- | :-- | :-- | :-- |
-| `Invoiceable` | boolean | required | Whether the company is invoiceable or not. |
-| `AddFeesToInvoices` | boolean | required | Whether the company has an additional fee applied for invoicing or not. |
+`[PlatformAddress]/api/connector/v1/companies/getAllByName`
 
-#### Credit rating
+```javascript
+{
+  "Client": "string",
+  "LanguageCode": "string",
+  "CultureCode": "string",
+  "AccessToken": "string",
+  "ClientToken": "string",
+  "Name": "string"
+}
+```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Basic` | string [Credit rating basic](#credit-rating-basic)| optional | The level of creditworthiness of the company. |
+| `ClientToken` | string | required |  |
+| `AccessToken` | string | required |  |
+| `Client` | string | required |  |
```

Comment:
❗ ToDo: figure out why `Client`, `AccessToken` and `ClientToken` description is not automatically generated

Reply:
yep, that solved it 👍